web/gui-v2/src/components/CellStat.jsx

```diff
@@ -23,13 +27,50 @@ const styles = {
       color: #a0a0a0;
     }
   `,
+  cellWithTooltip: css`
+    /* Add a column for the tooltip to fit in.  Same note as above on 3.2em. */
+    grid-template-columns: auto 1fr 3.2em;
+  `,
 };
 
 const EMPTY_VALUES = [null, '---'];
 
-const CellStat = ({data, colKey}) => {
+const tooltipTypes = {
+  "United States": {},
+  "China": {
+    text: tooltips.jobsExplanations.chinaJobs,
+    Icon: WarningIcon,
+    iconCss: css`fill: var(--orange);`,
+  },
+  "noData": {
+    text: tooltips.jobsExplanations.noData,
+    iconCss: css`fill: var(--grey);`,
+  },
+  "other": {
+    text: tooltips.jobsExplanations.otherJobs,
+    Icon: WarningIcon,
+    iconCss: css`fill: var(--orange);`,
+  },
+};
+
+const CellStat = ({
+  col,
+  country = undefined,
+  data,
+}) => {
+  const tooltipType = (country === "United States") ? undefined : (
```

Comment:
This is a little bit hard to read, might be easier as a series of if statements!